libethereum/Client.cpp

```diff
@@ -654,7 +656,7 @@ size_t Client::syncTransactions(
     RevertableFSPatch::lastBlockTimestamp = blockChain().info().timestamp();
     StorageDestructionPatch::lastBlockTimestamp = blockChain().info().timestamp();
     POWCheckPatch::lastBlockTimestamp = blockChain().info().timestamp();
-
+    SkipInvalidTransactionsPatch::setLastBlock( blockChain().info() );
```

Comment:
Please follow the logic of other patches  above - the code needs to be consistent

Reply:
It seems bad idea to stick just to timestamps - in general, patches can have different logic of activation. E.g. see AmsterdamFixPatch.